types/tizen-tv-webapis/index.d.ts

```diff
@@ -0,0 +1,61 @@
+// Type definitions for @tizentv/tizen-tv-webapis 1.0
+// Project: https://developer.samsung.com/smarttv/develop/api-references/samsung-product-api-references.html
```

Comment:
Your package location is incorrect.
It needs to be  `tizentv__tizen-tv-webapis`

Reply:
I change to `tizen-tv-webapis`.

Reply:
`https://unpkg.com/browse/tizen-tv-webapis@1.0.1/types/` is this the package you are refering to? If yes it already contains types folder why not publish the d.ts. files right there?

Reply:
What is the `unpkg`? Her is my package[tizen-tv-webapis](https://www.npmjs.com/package/tizen-tv-webapis).

Reply:
@sheetalkamat Because I want to support developer to commonjs and typescript. The `tizen-tv-webapis` is for commonjs and the `@types/tizen-tv-webapis` is for typescript. If developers want to call tizen API, they include `js` file with `<script>` tag. So we support both style. 
 You may wonder why it include 'index.d.ts'.  The `ambient enum` is not include in compile time. But we want to support importing enum through `import`. Refer to `types/tizen-tv-webapis/tvinfo.d.ts`, there are horrible object. The object have 100 more key and value pairs. And it hard to fix it. Because we should support compatibility for legacy platform. So I thought using enum is better way to developers. I build the `ts` to `js` that include enum object. Finally that's `index.d.ts` is for build ts to js.

Reply:
@sheetalkamat Refer to 1.0.2, I removed the `index.d.ts`